blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/ExtensionClassContainerImpl.java

```diff
@@ -67,11 +69,9 @@ public Link getLink() {
 
     private Class getClazz(String name){
         try {
-            return this.getClass().getClassLoader().loadClass(name);
+            return Jenkins.getInstance().getPluginManager().uberClassLoader.loadClass(name);
```

Comment:
@vivek @imeredith I'm not sure how this was working before (my entire BO app was broken), it shouldn't have been AFAICT, please let me know if this is not the right thing to do for some reason.